### ISSUE TITLE
Slayer's Plate

### ISSUE THREAD

**Issue by kelsioboy** [magarena/magarena#889]
name=Slayer's Plate
image=http://magiccards.info/scans/en/soi/264.jpg
value=2.500
rarity=R
type=Artifact
subtype=Equipment
cost={3}
ability=Equipped creature gets +4/+2.;\
        Equip {3}
timing=equipment
requires_groovy_code
oracle=Equipped creature gets +4/+2.\nWhenever equipped creature dies, if it was a Human, put a 1/1 white Spirit creature token with flying onto the battlefield.\nEquip {3}

def effect = MagicRuleEventAction.create("Put a 1/1 white Spirit creature token with flying onto the battlefield.")

[
    new OtherDiesTrigger() {
        @Override
        public MagicEvent executeTrigger(final MagicGame game, final MagicPermanent permanent, final MagicPermanent died) {
            final MagicPermanent equipped = permanent.getEquippedCreature();
            return (equipped == died && equipped.hasSubType(MagicSubType.Human)) ?
                effect.getEvent(permanent) :
                MagicEvent.NONE;
        }
    }
]

**Comment by ShawnieBoy** [comment]
The trigger should return an event to use the stack:
```
[
    new OtherDiesTrigger() {
        @Override
        public MagicEvent executeTrigger(final MagicGame game,final MagicPermanent permanent,final MagicPermanent died) {
            final MagicPermanent equipped = permanent.getEquippedCreature();
            return (equipped == died && equipped.hasSubType(MagicSubType.Human)) ?
                new MagicEvent(
                    permanent,
                    this,
                    "PN puts a 1/1 white Spirit creature token onto the battlefield."
                ):
                MagicEvent.NONE;
        }
        @Override
        public void executeEvent(final MagicGame game, final MagicEvent event) {
            game.doAction(newPlayTokenAction(
                event.getPlayer(),
                CardDefinitions.getToken("1/1 white Spirit creature token with flying")
            ));
        }
    }
]
```
Using three ` characters can define a block of text 😉

**Comment by kelsioboy** [comment]
+1:

**Comment by kelsioboy** [comment]
:+1:

**Comment by melvinzhang** [comment]
@kelsioboy I included this on Sept 15 with 93ccb5c3ca5e3406c0. Please stick to this way of using MagicRuleEventAction.create, we would prefer to avoid the use of groovy code.

@ShawnieBoy Do note that it is preferable to use MagicRuleEventAction.create instead of writing out the groovy code so that we can make use of MagicRuleEventAction for consistency. The original code DOES return an effect via the expression `effect.getEvent(permanent)`
